src/index.ts

```diff
@@ -102,6 +103,27 @@ app.get("/shop", async (req: any, res: any) => {
     pages: Math.ceil(count/limit),
   });
 });
+
+app.get("/issues", async (req: any, res: any) => {
```

Comment:
There are type definitions for express route handler. You can replace `async (req: any, res: any)` with `async (req: Request, res: Response)`.

Before make sure you imported the from express `import { Request, Response } from "express"`

By doing this, you will benefit from autocompletion when you try to access request/response object properties.